assessment.rb

```diff
@@ -1,12 +1,70 @@
 require 'csv'
-# Assessment answers to:
+# question 1:
+
+class CreateLicensees < ActiveRecord::Migration
+  def change
+    create_table :licensee do |t|
+      t.string :surname
+      t.string :given_name
+      t.string :gender
+      t.integer :height
+      t.integer :weight
+      t.date :dob
+      t.string :eye_color
+      t.string :hair_color
+
+      t.timestamps null: false
+    end
+  end
+end
 
 # question 2:
 
+licensee = Licensee.create(surname: 'Bishop', given_name: 'Caren', gender: 'f', height: 63, weight: 131, dob: '1943-09-26', eye_color: 'Brown', hair_color: 'Black')
+p licensee
+
 # question 3:
 
+namespace :db do
+  namespace :populate do
+    desc 'fill the licensee table with example data'
+    task people: :environment do
+      Licensee.transaction do
+        CSV.foreach(Rails.root + 'data/licensee.csv',
+                    headers: true) do |licensee_row|
+          licensee = licensee_row.to_hash
+          next if Licensee.exists? licensee
+        Licensee.create! licensee
+          end
+        end
+      end
+    end
+  end
```

Comment:
Nice! It's wasn't expected that you would do this as a Rake task, but I'm pleased to see you getting more familiar with writing them.